src/main/java/io/epirus/console/project/ProjectCreatorConfig.java

```diff
@@ -36,4 +50,8 @@ public String getPackageName() {
     public String getOutputDir() {
         return outputDir;
     }
+
+    public Boolean getGenerateJar() {
```

Comment:
I think this would can be renamed to withJar or hasJar

Reply:
true